src_docs/developer/07.MannerOfThePullRequest.md

```diff
@@ -0,0 +1,79 @@
+# Manner of the Pull Request
+
+## Making Effective Comments on Pull Requests
+
+Pull Requests (PRs) are not just about submitting code — they are also a space for discussion and collaboration. Writing clear, relevant, and respectful comments is essential to keep development productive and enjoyable for everyone.
+
+This section is especially helpful for project managers, testers, and new contributors who want to provide feedback but are unsure how to do it effectively.
+
+---
+
+### 🎯 Stay Focused on the Scope
+
+Each PR typically aims to fix a specific problem or introduce a clearly defined feature. When reviewing a PR, please:
+
+- **Read the PR description** carefully. It explains the purpose and scope.
+- **Comment only on things directly related** to that purpose.
+- If you discover an unrelated issue, note it down elsewhere (e.g., in a new ticket or in a chat), rather than posting it in the PR.
```

Comment:
Sometimes people add off-topic comments to a pull request because they get new ideas while reading it. These ideas might feel related to them, but they are not directly connected to the purpose of the pull request. To keep things clear and focused, it's better to discuss those ideas in a new issue or pull request.

```suggestion
- If you discover an unrelated issue, note it down elsewhere (e.g., in a new issue ticket or in a development mail list), rather than posting it in the PR.
- Even if an idea feels related because it came up while reading the PR, if it’s not directly part of the change being discussed, please keep it separate by opening a new issue or starting a new discussion.
```